SevenSlots/SevenSlots/SevenSlots/Repository/UserRepository.cs

```diff
@@ -5,7 +5,7 @@
 using System.Text;
 using System.Text.Json;
 using System.Threading.Tasks;
-using Xamarin.Android.Net;
+//using Xamarin.Android.Net;
```

Comment:
Why did you comment this import instead of just removing it?

Reply:
Meant to ask you about this import, but then I saw the program working without them as well and just forgot about them. I'll delete them.